lua/apple-music/init.lua

```diff
@@ -44,6 +44,12 @@ local execute = function(cmd)
 	return pcall(exe, cmd)
 end
 
+local grab_os_version = function()
+	local command = [[ osascript -e 'set osver to system version of (system info)' ]]
+	local _, result = execute(command)
+	return tonumber(result)
```

Comment:
A version in x.y.z format can break this.

Reply:
Good catch! 
We only need the major version.

Reply:
Done.